src/v2/Apps/Purchase/Components/PurchaseHistory.tsx

```diff
@@ -127,10 +143,19 @@ const PurchaseHistory: React.FC<PurchaseHistoryProps> = (
   const pageInfo = me.orders.pageInfo
   const myOrders = me.orders.edges && me.orders.edges.map(x => x.node)
   return !loading ? (
-    <Box px={1}>
-      <Serif size="5">Purchases</Serif>
+    <Box>
+      <Sans size="6" px="10px" py="15px">
```

Comment:
```suggestion
      <Sans size="6" px={1} py="15px">
```

Not blocking, but it'd be good to revisit these 15px spacings. I wonder if 10px or 20px could be used instead (to keep w/ our spacing system).

Reply:
I'll ask Will. I used px everywhere for consistency

Reply:
We generally avoid 15px-like values throughout and rounded up or down to our design system so that we don't introduce design drift. 

Also second @zephraph in that we don't need `px` as that breaks consistency across the rest of our UI code; `styled-system` maps pixels to our [Theme](https://github.com/artsy/palette/blob/master/packages/palette/src/Theme.tsx#L155-L177).

Reply:
There's a [conversation](https://artsy.slack.com/archives/C9YNS4X32/p1600103909242000) about potentially introduction `1.5` b/c design has been using that. We can take that conversation to #design-systems

Reply:
👍 makes sense. 

(We should still replace all of the `px` references before merging, however, in favor of `1` `2`, etc.)